packages/components/app/styles/components/table.scss

```diff
@@ -103,6 +103,12 @@ $hds-table-cell-padding-tall: 22px 16px 21px 16px; // the 1px difference is to a
   }
 }
 
+// multi-select (isSelectable=true)
+
+.hds-table__th--is-selectable {
+  width: 48px;
```

Comment:
@didoo I changed the width to 48px to match the width of the checkbox. I set it to 1px before so that it would "shrink-wrap" the content without having to set an exact width (as the comment I had said.)

If I remove this width, then the table cell expands to be wider than the content. (I don't know what you were doing exactly when you looked at it.)

Reply:
That's OK with 48px.

Reply:
Ok well I preferred 1px since then it doesn't matter what size the checkbox is since the cell would always tightly wrap it but if it causes confusion, 48px is ok.